app/views/users/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1><%= @user.name %>'s Dashboard</h1>
+
+<%= button_to "Discover Movies", "/users/#{@user.id}/discover", method: :get %>
```

Comment:
I have this as a rails route for the next user story do you want to push it up this way or change it to a rails route now?